examples/body-to-query-params-server/app.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+app.use(bodyParser.json())
+const axios = require('axios');
+const { json } = require('stream/consumers');
+
+const port = 3000
+
+app.post('/', (req, res) => {
+    const headerOutboundUrl = req.get('Outbound-Url');
+    console.log('Outbount Url: ' + process.env.OUTBOUND_URL);
+    const outboundUrl = headerOutboundUrl + "?";
+
+    const outboundQueryParams = req.get('Query-Param-Fields');
+    console.log('Query Parameter Fields: ' + outboundQueryParams);
+
+    const params = outboundQueryParams.split(',');
+
+    console.log('POST body: ', req.body);
+
+    let outboundUrlThisRequest = outboundUrl;
+    params.forEach(param => {
```

Comment:
make sure you trim the param to remove any preceding spaces if they are set that way. for example, if someone does `Query-Param-Fields: foo, bar`